build/ninja-rules-cpp.sh

```diff
@@ -330,11 +330,22 @@ compile_one() {
         # Must have external 'time', and it must have -f
         # Notes: OS X has no time -f
         #        busybox time supports -f but not --format.
-        if command time -f '%e %M' true >/dev/null; then
-          set -- \
-            time -f "$out { elapsed: %e, max_RSS: %M }" -- \
-            "$@"
-        fi
+        case ${BASH_VERSION:-} in
+          3*)
+            # bash 3.2.x on macos has a bizarre bug that leads to weird
+            # interactions between errexit and `command time -f`. don't bother
+            # timing in this case.
+            ;;
+          *)
+            set +o errexit
```

Comment:
Do we still need to disable `errexit` ?  That might have been a red herring?

Reply:
Oh good question. Let me try on a newer version a bash

Reply:
Yup. Checks out. We don't actually need this. Will update